src/main/java/Event.java

```diff
@@ -0,0 +1,33 @@
+import java.text.ParseException;
+import java.util.Date;
+
+public class Event extends Task {
```

Comment:
Again, I really like that your Deadline and Event class are inherited from the main Task class. Good choice!